salt/metalk8s/orchestrate/upgrade/etcd.sls

```diff
@@ -0,0 +1,52 @@
+{%- set dest_version = pillar.orchestrate.dest_version %}
```

Comment:
I think we should run like this but that can't be a strong requirement IMO

Reply:
Disagree - it's a strong requirement, we cannot guarantee future versions won't need specific behaviour. And it's not that much of a disturbance for the end-user, he just needs to use the "good" Salt env, which he already has available

Reply:
Yes agree with that but ok lets do this in another state because we need several checks like the fact that we only upgrade from one version, we need to check the version of every nodes but it's maybe more part of the global upgrade runner #934

Reply:
Yes, let's do it in another ticket :)